### ISSUE TITLE
Comment Issue

### ISSUE THREAD

**Issue by Jeff-Thompson12** [pharmaR/risk_assessment#235]
Submitting a comment fails is `'` is included.
![image](https://user-images.githubusercontent.com/85879620/160924691-f8348eff-a325-4c44-a94a-3b3ecec90e0f.png)

**Comment by Jeff-Thompson12** [comment]
Alright, I see that the code contains the comment:
```
#' TODO: comments can't contain "'". Check for other invalid
#' characters.
```
So this is at least a known issue although I do not see an open issue on Github.

**Comment by Jeff-Thompson12** [comment]
Fixed by #236
